.github/workflows/ci.yaml

```diff
@@ -10,6 +10,42 @@ on:
     branches:
       - master
 jobs:
+  # uncheck-integration-test:
+  #   name: Mark integration test as not run
+  #   runs-on: ubuntu-20.04
+  #   steps:
+  #     - id: get-pr-body
+  #       name: Get the current PR body
+  #       uses: jwalton/gh-find-current-pr@v1
+  #       with:
+  #         state: open
+          
+  #     - id: create-unchecked-pr-body
+  #       name: Create unchecked PR body
+  #       run: |-
+  #         UNCHECKED_BODY=$(sed 's/- \[[Xx]\] Started end-to-end tests on the latest commit/- \[ \] Started end-to-end tests on the latest commit/' <<\EOF
+  #         ${{ steps.get-pr-body.outputs.body }}
+  #         EOF
```

Comment:
did you comment this out for debugging purposes?

Reply:
yup!